DfE.FindInformationAcademiesTrusts/Pages/Trusts/Academies/_OfstedRatingCell.cshtml.cs

```diff
@@ -1,35 +1,48 @@
+using System.ComponentModel.DataAnnotations;
+using DfE.FindInformationAcademiesTrusts.Data;
+using DfE.FindInformationAcademiesTrusts.Extensions;
+
 namespace DfE.FindInformationAcademiesTrusts.Pages.Trusts.Academies;
 
-internal static class OfstedRatingConstants
+public enum OfstedRatingValues
 {
-    public const string NotYetInspected = "Not yet inspected";
+    [Display(Name = "Not yet inspected")] None = -1,
+    [Display(Name = "Outstanding")] Outstanding = 1,
+    [Display(Name = "Good")] Good = 2,
+
+    [Display(Name = "Requires improvement")]
+    RequiresImprovement = 3,
+    [Display(Name = "Inadequate")] Inadequate = 4
 }
 
 public class OfstedRatingCellModel
 {
     public required DateTime? AcademyJoinedDate { get; init; }
-    public required string Rating { get; init; }
-    public required DateTime? RatingDate { get; init; }
 
-    public bool HasRating()
-    {
-        return Rating != OfstedRatingConstants.NotYetInspected;
-    }
+    public required OfstedRating OfstedRating { get; init; }
 
-    public bool IsAfterJoining()
-    {
-        return RatingDate >= AcademyJoinedDate;
-    }
+    public bool IsAfterJoining => OfstedRating.InspectionEndDate >= AcademyJoinedDate;
 
-    public string GetTagClasses()
+    public string? OfstedRatingDescription => ((OfstedRatingValues)OfstedRating.OfstedRatingScore).GetDisplayName();
```

Comment:
This would be nicer as a switch expression rather than using a special display enum